addons/mail/static/src/components/composer_text_input/composer_text_input.xml

```diff
@@ -11,6 +11,7 @@
                     />
                 </t>
                 <textarea class="o_ComposerTextInput_textarea" t-att-class="{ 'o-composer-is-compact': props.isCompact }" style="height: 0;" t-esc="composer.textInputContent" t-att-placeholder="textareaPlaceholder" t-on-focusin="_onFocusinTextarea" t-on-focusout="_onFocusoutTextarea" t-on-keydown="_onKeydownTextarea" t-on-keyup="_onKeyupTextarea" t-on-input="_onInputTextarea" t-ref="textarea"/>
+                <textarea class="o_ComposerTextInput_textarea_dummy" t-att-class="{ 'o-composer-is-compact': props.isCompact }" style="height: 0;" t-esc="composer.textInputContent" t-ref="textarea_dummy" disabled="1" />
```

Comment:
Is it necessary/safe to set disabled=1? Any attribute that differs from the original textarea might influence the way it is rendered.

Also not having the class `o_ComposerTextInput_textarea` is problematic because it has style scss that has to be taken into account in the compte.

Reply:
The disable is to avoid selecting the textarea with `tab`. We don't really want this textarea edited at all by anything else than javascript.
I have add the `o_ComposerTextInput_textarea` class

Reply:
Ok if disable is to prevent tab that sounds important. Let's hope it doesn't change style in a way that would break height computation.